ccmlib/common.py

```diff
@@ -461,6 +461,37 @@ def scylla_extract_install_dir_and_mode(install_dir):
     return install_dir, scylla_mode
 
 
+def wait_for(func, timeout, first=0.0, step=1.0, text=None):
+    """
+    Wait until func() evaluates to True.
+
+    If func() evaluates to True before timeout expires, return the
+    value of func(). Otherwise return None.
+
+    :param func: Function that will be evaluated.
+    :param timeout: Timeout in seconds
+    :param first: Time to sleep before first attempt
```

Comment:
Please say "time (in seconds) to sleep", just like you said above "timeout in seconds". Seconds is probably the natural choice, but better be explicit about it.

Reply:
Done

Reply:
I don't see it done, maybe you forgot to push?